CHANGELOG.md

```diff
@@ -1,8 +1,14 @@
 # Changelog
 
-## libR-sys (unreleased)
+## libR-sys 0.5.0
 
+- Update the default precomuted bindings to R 4.3.0.
+- Remove deprecated API `VECTOR_PTR` from bindings [[#165]](https://github.com/extendr/libR-sys/pull/165)
+- Use Rust constants (i.e., `::std::os::raw::c_uint` is now replaced with `u32`) [[#160]](https://github.com/extendr/libR-sys/pull/160)
```

Comment:
Use rust types instead of c-types. I snuck that in there because R don't use exotic types across all the supported platforms. I think.. And thus I snuck it in there with the constants removal.

Reply:
That's much better than whatever I wrote. Perfect!

Reply:
Thanks!